Gridicons/Gridicons.xcodeproj/project.pbxproj

```diff
@@ -282,7 +283,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
WP is still at iOS 11.0. Will this conflict?

Reply:
Freudian slip. I'm in denial that WPiOS still supports iOS 11 😛